.gitignore

```diff
@@ -67,6 +67,7 @@ gridstatus/tests/fixtures/isone/vcr_cassettes
 gridstatus/tests/fixtures/pjm/vcr_cassettes
 gridstatus/tests/fixtures/ercot_api/vcr_cassettes
 gridstatus/tests/fixtures/ieso/vcr_cassettes
+gridstatus/tests/fixtures/isone_api/vcr_cassettes
```

Comment:
Would it make more sense to use `**/vcr_cassettes` for now? Seems like individually listing the paths is more error prone.

Reply:
agreed, updated.